lib/api/graph.js

```diff
@@ -108,13 +108,69 @@ function createEmbed(graphJSON, options, callback){
     client.request("POST", "/graph/embed", params, callback);
 }
 
+/*section: graph
+ *comment: delete an embed with a specific id
+ *params:
+ *  embedId: the id of the embed to delete
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var embedid = "foo";
+ *  dogapi.graph.deleteEmbed(embedid, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function revokeEmbed(embedId, callback){
+    client.request("GET", "/graph/embed/" + embedId + "/revoke", callback);
+}
+
+
+/*section: graph
+ *comment: get all embeds from datadog
+ *params:
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  dogapi.graph.getEmbeds(function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function getEmbeds(callback) {
+    client.request("GET", '/graph/embed', callback);
+}
+
+/*section: graph
+ *comment: get a single embed
+ *params:
+ *  embedId: the id of the embed to get
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var embedId = "foo";
+ *  dogapi.graph.getEmbed(embedId, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function getEmbed(embedId, callback) {
+    client.request("GET", '/graph/embed/' + embedId, callback);
+}
+
 module.exports = {
     snapshot: snapshot,
     createEmbed: createEmbed,
+    revokeEmbed: revokeEmbed,
+    getEmbeds: getEmbeds,
+    getEmbed: getEmbed,
     getUsage: function(){
         return [
             "  dogapi graph snapshot <query> <from> <to> [--events <event-query>]",
-            "  dogapi graph create_embed <graph_json> [--timeframe <timeframe>] [--size <size>] [--legend <legend>] [--title <title>]"
+            "  dogapi graph create_embed <graph_json> [--timeframe <timeframe>] [--size <size>] [--legend <legend>] [--title <title>]",
+            "  dogapi graph revoke_embed <graph_id>",
+            "  dogapi graph get <graph_id>",
+            "  dogapi graph getall <graph_id>"
```

Comment:
although, we should probably have these be `get_embed` and `get_embeds` to be consistent with the above methods.